builtin/plugins/chainconfig/chainconfig.go

```diff
@@ -97,6 +109,10 @@ func featureKey(featureName string) []byte {
 	return util.PrefixKey([]byte(featurePrefix), []byte(featureName))
 }
 
+func ActionKey(actionName string) []byte {
```

Comment:
`ActionKey` -> `actionKey`

Reply:
It is used outside the package. So I need to export it.

Reply:
Done